src/utils/CodecParser.h

```diff
@@ -0,0 +1,54 @@
+/*
+ *  Copyright (C) 2023 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include <bento4/Ap4Ac3Parser.h>
+#include <bento4/Ap4AdtsParser.h>
+#include <bento4/Ap4Eac3Parser.h>
+

```

Comment:
what do you think to move these new files (CodecParser.*) to `/parser` folder instead of utils?
and also suggest to wrap all header code with an appropriate namespace